tests/test_all.py

```diff
@@ -351,7 +359,7 @@ def test_no_orphaned_packages(container_per_test: ContainerData) -> None:
         "sle-module-basesystem-release",
         "sle-module-python3-release",
         "sle-module-server-applications-release",
-    }
+    } + monitoring_stack_packages
```

Comment:
`+` is not supported for sets
```suggestion
    }.union(monitoring_stack_packages)
```